ext/standard/tests/streams/stream_socket_sendto.phpt

```diff
@@ -0,0 +1,71 @@
+--TEST--
+int stream_socket_sendto ( resource $socket , string $data [, int $flags = 0 [, string $address ]] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
```

Comment:
This should be removed, like in your other PRs, no need to check for PHP versions